openlibrary/templates/account/reading_log.html

```diff
@@ -44,8 +44,7 @@
   $if len(docs) > 0:
     $if key in readlog_keys:
       <form method="GET" class="olform pagesearchbox">
-        <input type="text" minlength="3" placeholder="$_('Search your reading log')" name="q" value="$(query_param('q', ''))"/>
-        <input type="submit"/>
+        $:render_template("search/searchbox", q=query_param('q'), placeholder='Search your reading log')
```

Comment:
```suggestion
        $:render_template("search/searchbox", q=query_param('q'), placeholder=_('Search your reading log'))
```
Use localized string.

Reply:
Sorry about that, the i18n part completely slipped from my mind. I'll commit the provided suggestions. Thanks for the help @jimchamp.